lib/Controller/WopiController.php

```diff
@@ -125,7 +126,8 @@ public function __construct(
 		UserScopeService $userScopeService,
 		FederationService $federationService,
 		IEncryptionManager $encryptionManager,
-		IGroupManager $groupManager
+		IGroupManager $groupManager,
+		IVersionManager $versionManager
```

Comment:
Unfortunately there is no public API (as the namespace is OCA instead of OCP) so we cannot use dependency injection here as the files_version app can be disabled by admins and then the class would not be available.

We'll need to stick for getting the class from the server container and catch the ContainerExceptionInterface just incase.

Reply:
Nice catch, fixed :)